lib/global.dart

```diff
@@ -35,3 +35,11 @@ class BaseEnv {
 }
 
 final BaseEnv baseEnv = BaseEnv();
+
+
+
+abstract class SharedPreferencesKeys {
+  static const isWalletCreated = 'IS_APP_INSTALLED';
```

Comment:
we're using those keys in a single class just, so we shouldn't need a global class with keys. Later on we will have abstraction arround shared prefs and secure storage, which will encapsulate the way we interact with those and keys can be private and local to that class only

Reply:
This key is used in two different classes `mnemonic_encryptor.dart` and `mnemonic_onboarding.dart`. I just created this small class so that we don't need to maintain the key in two different places. We can refactor this when we place an abstraction around this. Let me know how to use these keys in two classes if I remove this class from here. Should I use them directly as strings?

Reply:
that's absolutely fine then! I've wrongly assumed it's used currently in one place. We'll deal with that when we have abstraction arround shared prefs